source/_components/sensor.sensehat.markdown

```diff
@@ -34,5 +35,119 @@ Configuration variables:
   - 'temperature'
   - 'humidity'
   - 'pressure'
+  is_hat_attached (Optional): True|False boolean; True declares the SenseHAT _is_ physically on the Raspberry Pi
+  
 
+#### Customizing the Sense HAT data
```

Comment:
Yes, I know, but I also wrote back that the page was completely broken. That a section was missing and the order was wrong.
This is how the page should be rendered.
You're choice if you want to embed this one.